src/bentoml_cli/cloud.py

```diff
@@ -76,7 +76,11 @@ def login(shared_options: SharedOptions, endpoint: str, api_token: str) -> None:
             baseURL = f"{endpoint}/api_tokens"
             encodedCallback = urllib.parse.quote(callback_server.callback_url)
             authURL = f"{baseURL}?callback={encodedCallback}"
-            Confirm.ask(f"Press Enter to open [blue]{authURL}[/] in your browser...")
+            answer = Confirm.ask(
+                f"Press Enter to open [blue]{authURL}[/] in your browser..."
+            )
+            if not answer:
+                return
```

Comment:
change it to

```python
if Confirm.ask(...):
    if webbrowser.open_new_tab(authURL):
        ...
    else:
        ...
try:
     code = callback_server.wait_indefinitely_for_code()
...
```
So even user answers no, he can open the URL in browser himself and bentoml can still respond to it.
That will cause a indefinte waiting so user has to press ctrl-c if he wants to quit

Reply:
Sounds Great!